cypress/integration/commit_queue.ts

```diff
@@ -59,4 +59,13 @@ describe("commit queue page", function() {
       cy.get("[data-cy=commitQueue-card-error]").contains(errorMessage);
     });
   });
+
+  it("Clicking on remove a patch from the commit queue should work", () => {
```

Comment:
what is meant to happen if there is an error removing the item?

Reply:
There arent any error states in the designs. But ideally if theres an error we would so some sort of universal error banner to indicate there was an error on the page

Reply:
Cant really write a test for an error if theres no error state.

Reply:
Just spoke to Lara and she showed me the error banners. If these arent implemented already. I can probably make them in a separate ticket and use them here.

Reply:
there is already a ticket for it and i'm working on it right now. a lot of things are not included in the designs that we as front end engineers must read between the lines for. how it looks is irrelevant. my question is about **what** should happen when there is an error because right now i do not believe anything in the ui will happen. you don't need to show a banner, just render an error div with a TODO for showing the banner.

Reply:
Nothing in the UI does happen currently but i will add a conditionally rendered block to display the error in the card ui. I did leave a comment in the component that says "Replace this with better error handling" https://github.com/evergreen-ci/spruce/pull/150/files#diff-1b4c4e95f91af6974a5c0ce1977feeaeR52

Reply:
a catch block is where we would log the error. all interactions should still handle loading/error on a ui level.